app/scripts/controllers/eventRegistrations.js

```diff
@@ -93,6 +93,50 @@ angular.module('confRegistrationWebApp')
       $scope.queryParameters.block = visibleBlocks;
     };
 
+    // Set initial Hide Column values
+    $scope.columns = ['Started', 'Completed', 'Email'];
+    $scope.columnChecked = [true, true, true];
+    $scope.showStarted = true;
+    $scope.showCompleted = true;
+    $scope.showEmail = true;
```

Comment:
Hey Scotty.


I think those are good ideas!

I can make the changes when I get back from vacation July 11th.

Will this be ok?


Doug



________________________________
From: Scotty Waggoner <notifications@github.com>
Sent: Thursday, June 30, 2016 3:41 PM
To: CruGlobal/conf-registration-web
Cc: Doug Worner; Mention
Subject: Re: [CruGlobal/conf-registration-web] EVENT-428 (#475)


In app/scripts/controllers/eventRegistrations.js<https://github.com/CruGlobal/conf-registration-web/pull/475#discussion_r69206403>:

> @@ -93,6 +93,50 @@ angular.module('confRegistrationWebApp')
>        $scope.queryParameters.block = visibleBlocks;
>      };
>
> +    // Set initial Hide Column values
> +    $scope.columns = ['Started', 'Completed', 'Email'];
> +    $scope.columnChecked = [true, true, true];
> +    $scope.showStarted = true;
> +    $scope.showCompleted = true;
> +    $scope.showEmail = true;


If you end up keeping this logic separate, I think you could keep it all in a single object such as:

var builtInColumnsVisible = {
    Email: true,
    Started: true,
    Completed: true
};


You could loop through the keys to get the column names (that would probably not be as useful when you have the custom columns in between) and look up the visible state by key in both the dropdown and in the table.

-
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/CruGlobal/conf-registration-web/pull/475/files/20aafdb8e9e3c550f2601a5291cc40beccbadc85#r69206403>, or mute the thread<https://github.com/notifications/unsubscribe/AQ17ehFssS1o5WpgVjLx1wlbU41IdQL2ks5qRCnogaJpZM4JBkdJ>.